Dockerfile

```diff
@@ -1,22 +1,19 @@
 FROM ubuntu:18.04
 LABEL maintainer="ehernandez@ioet.com"
 
-ARG DB_URI
-
-RUN apt-get update
-RUN apt-get install -y git
+RUN apt-get update --fix-missing
 RUN apt-get install -y python3 python3-dev python3-pip
 RUN apt-get install -y curl
 RUN apt-get install -y jq
 
```

Comment:
You can reduce the numbers of layers (and the image size) installing all the dependencies at once.
`RUN apt-get install -y <packageA> <packageB>`
https://hackernoon.com/tips-to-reduce-docker-image-sizes-876095da3b34